views/js/controller/runtime/testRunner.js

```diff
@@ -1078,6 +1106,9 @@ define([
                 TestRunner.beforeTransition();
                 TestRunner.testContext = testContext;
 
+                TestRunner.keepItemTimed(TestRunner.getSessionStateService().getDuration());
+                TestRunner.getSessionStateService().restart();
```

Comment:
I have a concern about this code, as the `keepItemTimed` will call the update method after the server has answered.

If you look the code above, there is already an explicit call to the update method, when the `onServiceApiReady` event is triggered.

If you trace the `update` method you will see that it is called twice on each page refresh. This is a waste of network resources, as this method triggers an item loading each time.

Maybe it can be better to place the call to the `keepItemTimed ` method in the `onServiceApiReady` event, instead of the `update` method. So this last will be called only once.